portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,21 +26,58 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import java.util.ArrayList; 
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.sps.data.Task;
+
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-/**
+  private int quantity = 5; 
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("data/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+
+    // Create a new query
+    Query query = new Query("Task");    
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    // Get query string
+    //int limit = Integer.parseInt(request.getQueryString()); 
+
+    // Set query limit to control maximum number of comments
+    List<Entity> sentence = results.asList(FetchOptions.Builder.withLimit(quantity));
```

Comment:
Why is a list of Entity a |sentence|?

Reply:
I chose sentence because my entities are stored as verb, animal, and adj, used to create a sentence